toml.md

```diff
@@ -417,8 +420,8 @@ str = ''''That,' she said, 'is still pointless.''''
 ```
 
 Control characters other than tab are not permitted in a literal string. Thus,
-for binary data, it is recommended that you use Base64 or another suitable ASCII
-or UTF-8 encoding. The handling of that encoding will be application-specific.
+for binary data, it is recommended that you use Base64 or another suitable text
+encoding. The handling of that encoding will be application-specific.
```

Comment:
I would change "text encoding" to "binary-to-text encoding", as Wikipedia [has it](https://en.wikipedia.org/wiki/Binary-to-text_encoding).

Reply:
Alright.